### DISCUSSION TITLE
YI Camera A136

### DISCUSSION THREAD

**Discussion by DVStan** [roleoroleo/yi-hack-MStar#430] [Q&A]
I think I might have bricked my YI Outdoor 1080P PTZ Camera - https://www.yitechnology.com/outdoor-ptz

The serial number starts with A136. I am looking for possible firmware or even stock firmware that I could use to unbrick the camera.

If anyone has any information or able to help it would be much appreciated. 

Thank you...

**Comment by roleoroleo** [comment]
Share some pictures of the board.

**Reply by DVStan** [reply]
Attached are the pictures of the board and camera as well as the ID code.
![20220318_134027](https://user-images.githubusercontent.com/101794651/158955796-bd8046dd-b61a-4282-951e-a3b5268028c9.jpg)
![20220223_172528](https://user-images.githubusercontent.com/101794651/158955828-6f6ac3d3-1f37-47fb-89cb-26b0b1d70c6c.jpg)
![20220318_134327](https://user-images.githubusercontent.com/101794651/158955837-863dfbd7-1a26-47e9-9629-ba815aae1333.jpg)

**Comment by roleoroleo** [comment]
Try to take a picture where I can read the model:
![immagine](https://user-images.githubusercontent.com/39277388/158975250-46ecb1dc-9af4-4a96-9fc8-c1a4a69e9c95.png)

**Reply by DVStan** [reply]
That is part of the issue. There is no place anywhere that I can find a model number. Is there a IC chip that I can take a close up picture of that would help?

**Reply by roleoroleo** [reply]
I mean the model of the chip in the picture above.

**Reply by DVStan** [reply]
Sorry, hope this helps. The problem is that it does not boot. The yellow lights come on and nothing else happens. Tried the reset button many times. No change. Acts like the firmware is messed up.
![20220318_205604](https://user-images.githubusercontent.com/101794651/159007388-067a99d8-b816-4d33-be27-e080f88694ce.jpg)

**Comment by roleoroleo** [comment]
None of my cameras have this chip, sorry.

**Reply by DVStan** [reply]
Ok, thanks. Now I have no idea the brand / model / manufacturer to find the correct firmware or get this thing going. But thank you for your help and quick response.

**Comment by roleoroleo** [comment]
If you are able to dump the fw, I could try to analyze it.
Maybe yi uses the same "backdoor" for this cam:
https://github.com/roleoroleo/yi-hack-Allwinner-v2/wiki/Dump-your-backup-firmware-(SD-card)

**Reply by DVStan** [reply]
That would be much appreciated... I am trying to get the fw dump. But it does not seem to be booting of the sd card. I will get a different sd card and try again. I will let you know...

Thank you

**Reply by DVStan** [reply]
Well thank you for your patience roleoroleo. But I believe the camera is toast. It will not boot up, unable to get a dump, tried 2 different new SD cards and it will not read or boot to either to get a dump or load any firmware. Also the 6 yellow lights stay on and the system does nothing when I hold in the reset button.

Thank you for your help...
